web/cobrands/bristol/assets.js

```diff
@@ -19,6 +19,12 @@ var park_style = new OpenLayers.Style({
     fillOpacity: "0.25"
 });
 
+var park_style_map = new OpenLayers.StyleMap({
+    default: park_style,
+    select: park_style,
```

Comment:
You don't need to set select or hover on a layer that can't be selected or hovered :)